python/dgl/contrib/cugraph/nn/__init__.py

```diff
@@ -0,0 +1 @@
+from .conv import *
```

Comment:
We will  have to make this optional : 
```suggestion
try:
     from .conv import *
except ModuleNotFoundError:
        raise ModuleNotFoundError("dgl.contrib.cugraph.nn requires pylibcugraphops to be installed")
```